shell/platform/windows/flutter_windows.cc

```diff
@@ -76,6 +79,9 @@ FlutterDesktopViewControllerRef FlutterDesktopViewControllerCreate(
   // Must happen after engine is running.
```

Comment:
I don't understand this feedback, appols.

Reply:
I just realized this part is pure placeholder; I was saying that part of the code above was relevant to UWP even though not all of it was. But since there's no implementation at all yet, never mind.